Library/Homebrew/github_packages.rb

```diff
@@ -115,33 +120,42 @@ def upload_bottle(user, token, skopeo, formula_name, bottle_hash)
       ".#{rebuild}"
     end
     version_rebuild = "#{version}#{rebuild}"
-    root = Pathname("#{formula_name}-#{version_rebuild}")
+    root = Pathname("#{formula_name}--#{version_rebuild}")
     FileUtils.rm_rf root
 
     write_image_layout(root)
 
     blobs = root/"blobs/sha256"
     blobs.mkpath
 
-    formula_path = HOMEBREW_REPOSITORY/bottle_hash["formula"]["path"]
-    formula = Formulary.factory(formula_path)
-
     # TODO: ideally most/all of these attributes would be stored in the
     # bottle JSON rather than reading them from the formula.
     git_revision = formula.tap.git_head
     git_path = formula_path.to_s.delete_prefix("#{formula.tap.path}/")
     source = "https://github.com/#{org}/#{repo}/blob/#{git_revision}/#{git_path}"
+    documentation = if formula.tap.core_tap?
+      "https://formulae.brew.sh/formula/#{formula_name}"
```

Comment:
This feels a bit tenuous as "documentation"

Reply:
We could swap `org.opencontainers.image.documentation` and `org.opencontainers.image.url`. For me it hinges on whichever looks more useful in the GitHub Packages web interface rendering of the metadata.